### PR TITLE
Topic/1640 signature to outgoing emails v2

### PR DESCRIPTION
**Pull request by foxlynx**
#1640 

# Definition of Done / Review checklist

## Reviewability
- [ ] link to issue
- [ ] note if PR is on top of other PR
- [ ] note if related change in rems-deploy repo
- [ ] consider adding screenshots for ease of review

## API
- [ ] API is documented and shows up in Swagger UI
- [ ] API is backwards compatible or completely new
- [ ] Events are backwards compatible

## Documentation
- [ ] update changelog if necessary
- [ ] add or update docstrings for namespaces and functions
- [ ] components are added to guide page
- [ ] documentation _at least_ for config options (i.e. docs folder)
- [ ] ADR for major architectural decisions or experiments

## Different installations
- [ ] new configuration options added to rems-deploy repository
- [ ] instance specific translations (i.e. LBR kielivara)

## Testing
- [ ] complex logic is unit tested
- [ ] valuable features are integration / browser / acceptance tested automatically

## Accessibility
- [ ] all icons have the aria-label attribute
- [ ] all fields have a label
- [ ] errors are linked to fields with aria-describedby
- [ ] contrast is checked
- [ ] conscious decision about where to move focus after an action

## Follow-up
- [ ] new tasks are created for pending or remaining tasks
- [ ] no critical TODOs left to implement

### CODE REVIEW

File: resources/translations/fi.edn [hunk 1]
```diff
@@ -264,6 +264,7 @@
                          :message "Hyvä %1,\n\nSinut on lisätty käyttäjän %4 jättämän hakemuksen %3 jäseneksi.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot osoitteessa %6"}
           :member-invited {:subject "Kutsu osallistumaan hakemukseen %3"
                            :message "Hyvä %1,\n\nSinut on kutsuttu osallistumaan käyttäjän %2 jättämään käyttöoikeushakemukseen %3.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot osoitteessa %4"}
+          :regards "\n\nKind regards,\n\nREMS\n"
```
Comment (opqdonut) [inline resources/translations/fi.edn]: should be in finnish?

File: test/clj/rems/poller/test_email.clj [hunk 2]
```diff
@@ -7,6 +7,10 @@
             [rems.locales]
             [rems.poller.email :refer :all]))
 
+(defn empty-signature [f]
+  (with-redefs [rems.locales/translations (assoc-in rems.locales/translations [:en :t :email :regards] "")]
+    (f)))
```
Comment (opqdonut) [inline test/clj/rems/poller/test_email.clj]: did you consider also emptying the footer?

File: tests.edn [hunk 3]
```diff
@@ -16,5 +16,4 @@
                      :focus-meta [:focused]}]
             :kaocha/reporter kaocha.report/dots
             :plugins [:kaocha.plugin/print-invocations
-                      :kaocha.plugin/notifier
                       :kaocha.plugin/profiling]}
```
Comment (opqdonut) [inline tests.edn]: this will conflict with #1730

File: resources/translations/fi.edn (hunk 1 above)
Comment (foxlynx) [reply resources/translations/fi.edn]: True

**Comment by foxlynx** [comment]
Merging, requested changes done.

**Pull request merged**